app/scripts/directives/unbindService.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+(function() {
+
+  angular.module('openshiftConsole').component('unbindService', {
+    controller: [
+      '$filter',
+      'BindingModalUtils',
+      'DataService',
+      'ServiceInstancesService',
+      UnbindService
+    ],
+    controllerAs: 'ctrl',
+    bindings: {
+      target: '<',
+      bindings: '<',
+      applications: '<',
+      onClose: '<'
+    },
+    templateUrl: 'views/directives/unbind-service.html'
+  });
+
+  function UnbindService($filter, BindingModalUtils, DataService, ServiceInstancesService) {
+    var ctrl = this;
+    var context;
+    var getDisplayName = ServiceInstancesService.getDisplayName;
+    // var sortApplications = BindingModalUtils.sortApplications;
+    // TODO: once we have podPreset
+    // var deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets;
+
+
+    // TODO: once we have podPreset we will be able to associate the
```

Comment:
Eliminated this whole section, don't think any of it will ever be needed w/the maps from overview.